From d9234cca36319cb6895d4a7d36030a95923d1373 Mon Sep 17 00:00:00 2001
From: Vishesh Handa <me@vhanda.in>
Date: Fri, 15 Jun 2012 18:37:56 +0530
Subject: [PATCH] Remove a FIXME debug message

It's not a good idea to spam the users with FIXME errors.

BUG: 300323
---
 services/storage/resourcewatchermanager.cpp |    1 -
 1 file changed, 1 deletion(-)

diff --git a/nepomuk/services/storage/resourcewatchermanager.cpp b/nepomuk/services/storage/resourcewatchermanager.cpp
index 4ef5d6b..80883a4 100644
--- a/nepomuk/services/storage/resourcewatchermanager.cpp
+++ b/nepomuk/services/storage/resourcewatchermanager.cpp
@@ -120,7 +120,6 @@ Nepomuk2::ResourceWatcherManager::~ResourceWatcherManager()
 void Nepomuk::ResourceWatcherManager::addStatement(const Soprano::Statement& st)
 {
     // FIXME!
-    kError() << "FIXME";
     //addProperty( st.subject(), st.predicate().uri(), QList<Soprano::Node>() << st.object() );
 }
 
-- 
1.7.10.4

